src/Presentation/AmountFormatter.php

```diff
@@ -14,8 +14,8 @@ class AmountFormatter {
 	private $locale;
 
 	private $localeData = [
-		'de_DE' => [2, ',', '' ],
-		'en_US' => [2, '.', '' ]
+		'de_DE' => [2, ',', ''],
+		'en_US' => [2, '.', '']
```

Comment:
You could use Mediawiki custom sniff for it if really wanted to have PHPCS complaining. It also has a hidden ability to [remove unneeded elements from the array](https://phabricator.wikimedia.org/T134857)